src/generator.ts

```diff
@@ -116,7 +117,7 @@ export class Generator {
           { name: 'solid', message: 'Solid'  },
         ],
       },
-      !this.options.ct && {
+      !this.options.ct && !this.options['test-dir'] && {
```

Comment:
instead of removing the question with the boolean trick, you may want to consider using an initial value with the `initial` property and then skipping the question with the `skip` property:

- https://github.com/enquirer/enquirer/issues/364#issuecomment-1991338065

that would simplify the PR because the changes in the `_identifyChanges` method could be avoided

Reply:
Ok I see, it's much better. The `skip` option can be a boolean so I used that instead of a function.

Reply:
nice! looks much simpler